ngi_pipeline/database/filesystem.py

```diff
@@ -160,9 +166,18 @@ def create_charon_entries_from_project(project, best_practice_analysis="whole_ge
                                      'seqrun "{}" already exists; next...'.format(project, sample,
                                                                                   libprep, seqrun))
                     else:
+                        update_failed=True
                         LOG.error(e)
                         continue
 
+    if update_failed :
+        if retry_on_fail:
+        create_charon_entries_from_project(project, best_practice_analysis=best_practice_analysis,
+                                       sequencing_facility=sequencing_facility
+                                       force_overwrite=force_overwrite, delete_existing=delete_existing,
+                                       retry_on_fail=False):
```

Comment:
nice... recursive function with different parameters..... it should work but it is really a trick in a pretty tricky function.
what about using something like this https://julien.danjou.info/blog/2015/python-retrying